src/marqo/core/index_management/index_management.py

```diff
@@ -96,12 +98,17 @@ def bootstrap_vespa(self) -> bool:
         app = self.vespa_client.download_application()
         configured = self._marqo_config_exists(app)
 
+        query_profile_updated = self._update_query_profile(app)
```

Comment:
Just to clarify, when we deploy the vespa application, we're always reading from `search/query-profiles/default.xml`? Do we not also want to call `self._save_marqo_version(version.get_version())`

Reply:
It's my understanding the the _save_marqo_version is persisted after the initial configuration in the if block? I could be wrong